translations/messages/nl-be.yaml

```diff
@@ -5,6 +5,7 @@ messages:
     characters: "Karakters"
     deleted: "Verwijderd"
     upToDate: "Up-to-date"
+    archived: "Ge-archiveerd"
```

Comment:
One small thing, I think it's just 'gearchiveerd'.

Reply:
Haha you are right!